tensorflow/lite/micro/tools/ci_build/test_cortex_m_corstone_300.sh

```diff
@@ -43,13 +43,15 @@ readable_run make -f tensorflow/lite/micro/tools/make/Makefile clean
 readable_run make -j$(nproc) -f tensorflow/lite/micro/tools/make/Makefile CO_PROCESSOR=ethos_u OPTIMIZED_KERNEL_DIR=${OPTIMIZED_KERNEL_DIR} TARGET=${TARGET} TARGET_ARCH=${TARGET_ARCH} TOOLCHAIN=${TOOLCHAIN} build
 readable_run make -f tensorflow/lite/micro/tools/make/Makefile CO_PROCESSOR=ethos_u OPTIMIZED_KERNEL_DIR=${OPTIMIZED_KERNEL_DIR} TARGET=${TARGET} TARGET_ARCH=${TARGET_ARCH} TOOLCHAIN=${TOOLCHAIN} test
 
-# Run generic benchmark.
-readable_run make -j$(nproc) -f tensorflow/lite/micro/tools/make/Makefile \
-  CO_PROCESSOR=ethos_u \
-  OPTIMIZED_KERNEL_DIR=${OPTIMIZED_KERNEL_DIR} \
-  TARGET=${TARGET} \
-  TARGET_ARCH=${TARGET_ARCH} \
-  TOOLCHAIN=${TOOLCHAIN} \
-  GENERIC_BENCHMARK_MODEL_PATH=tensorflow/lite/micro/models/person_detect_vela.tflite \
-  GENERIC_BENCHMARK_ARENA_SIZE=`expr 150 \* 1024` \
-  run_tflm_benchmark
+# Run generic benchmark. Not supported for armclang - see comment in target makefile.
```

Comment:
@mansnils 

You mention a comment in the Generic Benchmark application makefile?  We would be very interested in hearing more about this issue.

Reply:
@mansnils 

Never mind, I found the makefile with the clang comment.  I now understand this change.